### DISCUSSION TITLE
随机挑个人送出minecraft国际版正版

### DISCUSSION THREAD

**Discussion by blqdswkl** [share121/inter-knot#1528] [其他]
恭喜你获得了minecraft正版！
本帖由我自己赞助！
下面是兑换代码！

114514-114514-114514-114514-191981-0HOMO1-919810
在这！不用谢！免费送！

**Comment by cpu644** [comment]
验证码有点过于臭了

**Comment by huxinchi** [comment]
好臭的验证码，不能要了！
